docs/components/table-view.md

```diff
@@ -0,0 +1,1016 @@
+# Table View
+
+Component for displaying data in table format.
+
+:::tip NOTE
+KTableView is a display-only component, it does not provide any data management capabilities like fetching, pagination, sorting or searching. If you are looking for a component that includes those functions out of the box, check out [KTable](/components/table).
```

Comment:
```suggestion
KTableView does not handle data management capabilities like data fetching, functional pagination, sorting or searching. If you are looking for a component that integrates with the data layer, check out [KTable](/components/table).
```

Doesn't KTableView handle sorting though?

Reply:
No, it only emits an event (can't handle sorting since `data` prop could only be accepting a paginated piece of otherwise larger dataset).